Makefile

```diff
@@ -10,16 +10,20 @@ HTTPS_GIT := https://github.com/axelarnetwork/axelar-core.git
 PUSH_DOCKER_IMAGE := true
 
 $(info $$WASM is [${WASM}])
+$(info $$IBC_WASM_HOOKS is [${IBC_WASM_HOOKS}])
 $(info $$CGO_ENABLED is [${CGO_ENABLED}])
 
 ifeq ($(WASM), true)
 WASM_ENABLED := "true"
+ifeq ($(IBC_WASM_HOOKS), true)
+IBC_WASM_HOOKS_ENABLED := "true"
```

Comment:
why not just always use `IBC_WASM_HOOKS_ENABLED` everywhere for the var name? That way you don't need this if statement

Reply:
yeah, we can update the check in app.go to expect `"true"` for the flag to enable it instead of just being non-empty (so that a setting of `"false"` works correctly)